data/json/snippets/zombie_anatomy.json

```diff
@@ -0,0 +1,89 @@
+[
+  {
+    "type": "snippet",
+    "category": "<zombie_harvest_general>",
+    "text": [
+      "As you make your first cut into its skin, a pressurized jet of decaying blood almost hits your face",
+      "The overbearing sweet smell of decay emanating from the carcass forces you to take short breaks during your work to not puke",
+      "As you cut through the muscle, it tenses up in reaction",
+      "You spot a few flies and maggots in its corpse, their chosen carcass not seeming to be very good for them, as most seem moribund",
+      "Random cavities in its body make you think it was atleast partially feeding on its own tissues to keep functioning",
+      "The corpse twitches as you begin.  it didn't feel like rigor mortis",
+      "The body stirrs as you open it up, suggesting its still trying to attack, and reminding you to be cautious",
+      "You begin your dissection - though, vivisection would be more appropriate, as you find the corpse not as still as you once thought"
```

Comment:
```suggestion
      "As you make your first cut into its skin, a pressurized jet of decaying blood almost hits your face.",
      "The overbearing sweet smell of decay emanating from the carcass forces you to take short breaks during your work to not puke.",
      "As you cut through the muscle, it tenses up in reaction.",
      "You spot a few flies and maggots in its corpse, their chosen carcass not seeming to be very good for them, as most seem moribund.",
      "Random cavities in its body make you think it was at least partially feeding on its own tissues to keep functioning.",
      "The corpse twitches as you begin.  It didn't feel like rigor mortis.",
      "The body stirs as you open it up, suggesting it's still trying to attack, and reminding you to be cautious.",
      "You begin your dissection - though, vivisection would be more appropriate, as you find the corpse not as still as you once thought."
```

Reply:
I can't accept this, dissection snippets do not have end punctuation because that end punctuation is supposed to be in the snippet template, which admittedly I still need to add. I will promptly commit the rest of your suggestion.

Reply:
Typo fixes applied, end punctuation rejected.

Reply:
>I can't accept this, dissection snippets do not have end punctuation

*json/snippets/harvest_drop_msgs.json intensifies*

Reply:
> > I can't accept this, dissection snippets do not have end punctuation
> 
> _json/snippets/harvest_drop_msgs.json intensifies_

You may notice from the adding composite snippet categories commit that this is now fixed, also I'm not even touching harvest_drop_msgs with this PR :D 

I'm not completely opposed to snippets having end punctuation, and I guess I could theoretically remove punctuation from the composite snippets and add specific punctuation to the existing snippets.
I mean I guess it would lead to more flexibility by allowing individual snippets to end in question, exclamation marks and triple points?
I'll ask Venera since they're the ones that added most of the mutant snippets why they did it that way.

Reply:
Got feedback from Venera, adding all your suggestions then removing punctuation from the composite snippets.